pkg/odo/cli/url/create.go

```diff
@@ -197,7 +219,25 @@ func (o *URLCreateOptions) Validate() (err error) {
 // Run contains the logic for the odo url create command
 func (o *URLCreateOptions) Run() (err error) {
 	if experimental.IsExperimentalModeEnabled() && util.CheckPathExists(o.DevfilePath) {
-		err = o.EnvSpecificInfo.SetConfiguration("url", envinfo.EnvInfoURL{Name: o.urlName, Port: o.componentPort, Host: o.host, Secure: o.secureURL, TLSSecret: o.tlsSecret})
+		if pushtarget.IsPushTargetDocker() {
+			for _, localURL := range o.EnvSpecificInfo.GetURL() {
+				if o.urlPort == localURL.Port && &localURL.ExposedPort != nil {
+					if !o.forceFlag {
+						if !ui.Proceed(fmt.Sprintf("Port %v already has an exposed port %v set for it. Do you want to override the exposed port", localURL.Port, localURL.ExposedPort)) {
+							log.Info("Aborted by the user")
+							return nil
```

Comment:
If the previous statement was Aborted by the user, shouldn't this be returning an error?

Reply:
IMO, if the port is already bound with an exposed port, this operation is just set a new exposed port. (similar to the set operation: https://github.com/openshift/odo/blob/f52328b9d4c996ba2c0647b6fb0bb91900f8a2bd/pkg/odo/cli/config/set.go#L125)
If user select not to overwrite, then the user knows there won't be any changes to the current env.yaml file. So it should not be an error.